packages/host/app/components/matrix/room-message.gts

```diff
@@ -93,13 +103,14 @@ export default class RoomMessage extends Component<Signature> {
   });
 
   <template>
+    <div class='hide' {{consumeContext consume=this.makeCardResources}} />
```

Comment:
I see there is one more place with this pattern of adding a hidden div and `.hide` style. Is this an opportunity to make a reusable component for this purpose?